cli-client/recognizer_stream.py

```diff
@@ -173,6 +174,20 @@ def wait_server(self) -> bool:
         
         return True
     
+    @staticmethod
+    def divide_audio(audio: bytes, chunk_size: int = 20000):
+        audioLength = len(audio)
+        nChunks = math.ceil(audioLength / chunk_size)
+        logging.debug("Dividing audio of length " + str(audioLength) + " into " + str(nChunks) + " of size " + str(chunk_size) + "...")
+        if nChunks > 1:
```

Comment:
You can remove the else if you only check for nChunks <= 1 and then yield audio. Then the method continues with the for and the field

Reply:
But it is not supposed to continue for nChunks/chunk_count <= 1